src/pages/index.js

```diff
@@ -209,6 +201,6 @@ function createCard(cardData, userId) {
 
 //Card Set Event Listeners
 addNewCardButton.addEventListener("click", () => {
-  addCardFormValidator.toggleButtonState();
+  formValidators["add-card-form"].resetValidation();
```

Comment:
Everything is correct here but...


* I can't open the modals with a form. See the errors http://joxi.ru/4Ak65neTVeQzV2 . It's because you need to add attribute `name` for any tag `form` in your `index.html`



So, for the profile form in line 102 you'll have this markup


```html

        <form class="modal__form" name="add-card-form" id="add-card-form">

```


After that it will work correctly